### ISSUE TITLE
New quest: tag seating (bench) at bus stops

### ISSUE THREAD

**Issue by SearchingForDottedLines** [westnordost/StreetComplete#1079]
Add tagging to indicate if a bus stop has seating or not per: https://wiki.openstreetmap.org/wiki/Tag:highway=bus_stop#Bus_stop

### General

Affected tag(s) to be modified/added: bench=* on nodes where highway=bus_stop and bench=* is not set
Question asked: Does this bus stop have seating?
Valid responses: Yes | No

### Checklist

Checklist for quest suggestions (see guidelines):
- [ Yes ] 🚧 To be added tag is established and has a useful purpose
- [ Yes ] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [ Yes ] 🐿️ Easily answerable by everyone from the outside but a survey is necessary 
- [ Yes ] 💤 Not an overwhelming percentage of elements have the same answer (No spam)
- [ Yes ] 🕓 Applies to a reasonable number of elements (Worth the effort)

### Ideas for implementation

Not sure if the question is better phrased as:

     Natural language: Does this bus stop have seating?
     Tag: Does this bus stop have a bench?

Basic overpass link to candidate nodes:

http://overpass-turbo.eu/map.html?Q=%5Bout%3Ajson%5D%5Btimeout%3A25%5D%3B%0A(%0A%20%20node%5B%22highway%22%3D%22bus_stop%22%5D%5B!%22bench%22%5D(-33.897171451452394%2C151.0599946975708%2C-33.862665104853846%2C151.1052918434143)%3B%0A)%3B%0Aout%20body%3B%0A%3E%3B%0Aout%20skel%20qt%3B

Potential other candidates:

Current bus stop quests also consider public_transport=platform see https://wiki.openstreetmap.org/wiki/StreetComplete/Quests

Responses:

https://taginfo.openstreetmap.org/keys/bench#values

| Response | Tag to set |
| ------------- | ------------- |
| Yes | bench=yes  |
| No | bench=no  |

**Comment by matkoniecz** [comment]
Duplicate of #200

**Comment by SearchingForDottedLines** [comment]
#200 seems to focus on bins more so than benches, which seemed to be the focus of discussion in the comments, but yes there is an overlap.

**Comment by westnordost** [comment]
What about bus stops where the bench is mapped separately? Also, how is this information useful? Please give an example

**Comment by SearchingForDottedLines** [comment]
Primary driver is accessibility.
Folks with limited mobility rely on seats (wherever they may be) to take a break while travelling.

**Comment by rugk** [comment]
But does it really matter for them how many seats there are?

**Comment by SearchingForDottedLines** [comment]
Count of seats is likely of lower priority than the existence of seats so this quest #1079 is more important than the other one #1080

**Comment by westnordost** [comment]
Okay, so your point is that it's not actually about the bus stop, but about the bench. I.e. the information is (only) useful for people searching for somewhere to sit, it is only by coincidence that it is a bus stop?

**Comment by SearchingForDottedLines** [comment]
Spot on - it's a bench that happens to be at a bus stop as much as it is a feature of the bus stop.
Good tagging practice has the bench tags on the same node as the bus stop when they are part of a related entity, in the same way the shelter and tactile paving are also.
In the event the bench is on a separate node in close proximity to the bus stop there not much we can do about that, though that is no different to the shelter and/or tactile paving being setup as separate nodes to the bus stop.

**Comment by matkoniecz** [comment]
I would consider it as an unwanted double tagging.

**Comment by SearchingForDottedLines** [comment]
Hi @westnordost - my reading of the OSM wiki indicates it is correct to tag the bench on the bus stop node, exactly the same as the shelter and tactile paving tags are handled:
https://wiki.openstreetmap.org/wiki/Tag:highway=bus_stop#Bus_stop

**Comment by westnordost** [comment]
Well, I think this would need to be cleared up in the community first.

**Comment by rory** [comment]
Old people (who in some places have free bus transport), might not be able to stand for long, and would benefit from knowing there is (or isn't) a bench at a bus stop.

**Comment by SearchingForDottedLines** [comment]
Agreed @westnordost, intent is to tag the correctly setup nodes only.
Cleanup (merging) nodes is outside the scope of StreetComplete

**Comment by Quix0r** [comment]
Some benches are mounted beneath a shelter and that one has passenger information (schedule), ads (billboard) and is lit sometimes. Maybe also include those tags? I sometimes manually add them (OSM's website editor).

**Comment by westnordost** [comment]
I think this could be implemented. If the bus stop has a bench, it describes a feature of that bus stop, independent of if that bench is mapped separately or not. Much as if a railway/metro stop is wheelchair enabled or not - elevators/ramps will be mapped separately, still it is useful information to have the wheelchair tag on the metro station as well. Same with `bridge=yes` tagged on ways going over `man_made=bridge` etc.

Also, of course, the wiki doesn't mention this topic at all. So the de-facto situation is that any data user interested in "places to sit" should search for a range of tags anyway (`leisure=picnic_table`, 'highway=steps', `barrier=rock`, ... bus stops with `bench=yes`) and not rely on that every bench should and is mapped as a separate feature.
